### PR TITLE
Buttons for 'adding person' and 'adding org'

### PR DESCRIPTION
**Pull request by allyr13**
I would like to get the buttons one on top of the other at some point and add some CSS styling but decided would be better to put into a different task

### CODE REVIEW

File: network/src/components/AppWrapper.js [hunk 1]
```diff
@@ -587,6 +587,30 @@ export const AppWrapper = ({children}) => {
...
+                  resetCanvasContextMenu();   // Reset the canvas context menu
+                  }} 
+                  style={{ padding: '10px 20px', 
+                    margin: '5px', 
+                    backgroundColor: 'blue', 
+                    color: 'white' 
+                  }}>
+                Add Organization
+              </button>
+            </div>
```
Comment (simon-dunk) [inline network/src/components/AppWrapper.js:656]: The functionality seems to be working correct here. However, the style looks a little odd with the placements of the buttons. I am not sure if this is how it is intended to look or perhaps it is different on my browser. 

image

**Review by simon-dunk — CHANGES_REQUESTED** [review]

File: network/src/components/AppWrapper.js (hunk 1 above)
Comment (allyr13) [reply network/src/components/AppWrapper.js:656]: This is how it is on my side. I added in the description for eventual CSS changes.

Comment (allyr13) [reply network/src/components/AppWrapper.js:656]: @simon-dunk Can we get this approved to just create a separate styling task?

**Review by simon-dunk — APPROVED** [review]

File: network/src/components/AppWrapper.js (hunk 1 above)
Comment (allyr13) [reply network/src/components/AppWrapper.js:656]: Task has been added to Asana

**Pull request merged**